media-sound/pasystray/pasystray-0.7.0.ebuild

```diff
@@ -0,0 +1,53 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+inherit gnome2-utils autotools vcs-snapshot xdg
```

Comment:
This is not a snapshot, why the use of `vcs-snapshot`?

Reply:
Many thanks for the review @a17r - specifically on this one:

inherit vcs-snapshot replaced a 'S=${WORKDIR}/${PN}-${P}' at one point because the tar ball (has always) extracts to portage/media-sound/pasystray-0.7.0/work/pasystray-pasystray-0.7.0 (rather than .../pasystray-0.7.0/work/pasystray-0.7.0) 

vcs-snapshot includes logic to just make it work. 

But I agree its confusing and I will revert to the more common S=${WORKDIR}/${PN}-${P}